spring-cloud-zuul-ratelimit-core/src/main/java/com/marcosbarbero/cloud/autoconfigure/zuul/ratelimit/config/properties/RateLimitType.java

```diff
@@ -83,6 +83,21 @@ public String key(HttpServletRequest request, Route route, RateLimitUtils rateLi
             return matcher;
         }
     },
+
+    /**
+     * Rate limit policy considering the HTTP request method.
+     */
+    METHOD {
+        @Override
+        public boolean apply(HttpServletRequest request, Route route, RateLimitUtils rateLimitUtils,/*not null*/ String matcher) {
+            return request.getMethod().equals(matcher.toUpperCase());
```

Comment:
Can you use `equalsIgnoreCase` instead of `matcher.toUpperCase`?

Reply:
ok